index.html

```diff
@@ -937,9 +947,58 @@ <h3><em>Detecting &amp; matching direction (TBD)</em></h3>
 
 <p class="reviewComments"><a href="https://github.com/w3c/i18n-activity/labels/t%3Abidi_detection_x" target="_blank">See related review comments.</a></p>
 </section>
-</section>
 
+<section id="strcat" class="subtopic">
+<h3>Concatenation of strings</h3>
 
+<p class="reviewComments"><a href="https://github.com/w3c/i18n-activity/labels/t%3Astrcat" target="_blank">See related review comments.</a></p>
+
+<div class="req" id="strcat_anti_pattern">
```

Comment:
I don't think people will find this general advice about composite messages, that has nothing to do with text direction.

I think it is best to move the section "Composite Strings" under 'Text-processing" and then add a 'see also' to the text direction section 3.6.

Reply:
pending telecon discuss